tmessage/cli.py

```diff
@@ -36,6 +38,21 @@ def on_message(client, userdata, message):
         print(Back.GREEN + Fore.BLACK + current_msg +
               Back.RESET + Fore.RESET + "")
 
+folder_name = 'messages'
+session_start_date = datetime.now().strftime('%Y-%m-%d_%H:%M')
+
+def store_messages(raw_msg):
+    try:
+        if not os.path.exists(folder_name):
+            os.mkdir(folder_name)
+    except:
+        pass

```

Comment:
No need to use `try` and `except` when you are using `if` to check for directory